source/elements/oneDAL/source/algorithms/decomposition/pca.rst

```diff
@@ -32,7 +32,31 @@ oneDAL specifies two methods for PCA computation:
 -----------------------
 Covariance-based method
 -----------------------
-[TBD]
+
+This method uses eigendecomposition of the covariance matrix to find the
+principal components of the dataset. The following steps are distinguished:
+
+#. Covariance computation
```

Comment:
Computation of covariance matrix

Computation of ...

Formation of the matrices storing the results

Reply:
fixed